src/app/state/transaction/types.ts

```diff
@@ -20,6 +20,11 @@ export enum TransactionType {
   BeaconPvssCommit = 'pvsscommit',
   BeaconPvssReveal = 'pvssreveal',
   BeaconVrfProve = 'vrfprove',
+  ConsensusDeposit = 'deposit',
+  ConsensusWithdraw = 'withdraw',
+  ConsensusAccountsParameters = 'parameters',
+  ConsensusBalance = 'balance',
+  ConsensusAccount = 'account',
```

Comment:
Rebase over https://github.com/oasisprotocol/oasis-wallet-web/pull/825/commits/7cc4cb1d0cd0e3bc709b3a9c93a1c4c497de961b

Reply:
I will also rebase on top of latest master as per @tjanez request.